hermes/templates/python/template.py.tpl

```diff
@@ -235,8 +235,8 @@ class DefaultSyntaxErrorHandler:
         return error
     def unexpected_eof(self):
         return self._error("Error: unexpected end of file")
-    def excess_tokens(self):
-        return self._error("Finished parsing without consuming all tokens.")
+    def excess_tokens(self, next_token):
+        return self._error("Finished parsing without consuming all tokens.  Next token was {next}".format(next=next_token))
```

Comment:
The one gotcha here (the reason why the tests are failing) is that the tests expect parsers in all languages to generate the same error messages.  Changing the message causes the tests to fail.  Let me think a little bit about how to make this better.  Definitely PyWDL should be implementing its own syntax error formatter class, but the function signatures need to be right